examples/resources/qdrant-cloud_accounts_role/resource.tf

```diff
@@ -0,0 +1,49 @@
+terraform {
+  required_version = ">= 1.7.0"
+  required_providers {
+    qdrant-cloud = {
+      source  = "qdrant/qdrant-cloud"
+      version = ">=1.1.0"
```

Comment:
Should the example point to a new version that contains the roles functionality?